spring-cloud-netflix-eureka-server/src/main/java/org/springframework/cloud/netflix/eureka/server/InstanceRegistryProperties.java

```diff
@@ -30,28 +30,28 @@
 
 	public static final String PREFIX = "eureka.instance.registry";
 
-
-	/* Default number of expected renews per minute, defaults to 1.
-	 * Setting expectedNumberOfRenewsPerMin to non-zero to ensure that even an isolated
+	/* Default number of expected client, defaults to 1.
+	 * Setting expectedNumberOfClientsSendingRenews to non-zero to ensure that even an isolated
 	 * server can adjust its eviction policy to the number of registrations (when it's
 	 * zero, even a successful registration won't reset the rate threshold in
 	 * InstanceRegistry.register()).
 	 */
-	@Value("${eureka.server.expectedNumberOfRenewsPerMin:1}") // for backwards compatibility
-	private int expectedNumberOfRenewsPerMin = 1;
+	@Value("${eureka.server.expectedNumberOfClientsSendingRenews:1}") // for backwards compatibility
```

Comment:
@spencergibb do you think we should support the old property as well as new property to avoid breaking people that transition to Greenwich?

Reply:
That would be nice since it does break others, like this: https://github.com/twinformatics/eureka-consul-adapter

Reply:
@maximusfloydus could you open a separate issue?

Reply:
https://github.com/spring-cloud/spring-cloud-netflix/issues/3286